test/scholar/metrics/discounted_cumulative_gain_test.exs

```diff
@@ -0,0 +1,44 @@
+defmodule Scholar.Metrics.DiscountedCumulativeGainTest do
+  use Scholar.Case, async: true
+  alias Scholar.Metrics.DiscountedCumulativeGain
+
+  describe "compute/2" do
+    test "computes DCG when there are no ties" do
+      y_true = Nx.tensor([3, 2, 3, 0, 1, 2])
+      y_score = Nx.tensor([3.0, 2.2, 3.5, 0.5, 1.0, 2.1])
+
+      result = DiscountedCumulativeGain.compute(y_true, y_score)
+
+      assert %Nx.Tensor{data: data} = Nx.broadcast(result, {1})
```

Comment:
You should assert here an expected value for particular data. For small tests, you can calculate the expected results by hand or use Sci-kit learn as a reference

Reply:
I agree, I'm going to use Sci-kit learn to check.